program-runtime/src/invoke_context.rs

```diff
@@ -1707,7 +1573,13 @@ mod tests {
             );
 
             assert!(result.is_ok());
-            assert_eq!(invoke_context.accounts_data_meter.remaining(), 0);
```

Comment:
The feature `enable_early_verification_of_account_modifications` is active in these tests, they would not pass otherwise because the `AccountsDataMeter` is no longer updated.